packages/arb-token-bridge-ui/src/components/Sidebar/AppSidebar.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+import dynamic from 'next/dynamic'
+import { usePostHog } from 'posthog-js/react'
+
+// Dynamically import the Sidebar component with SSR disabled
+const DynamicSidebar = dynamic(
+  () => import('@offchainlabs/cobalt').then(mod => ({ default: mod.Sidebar })),
+  { ssr: false }
+)
+
+export const AppSidebar = () => {
+  const posthog = usePostHog()
+  return (
+    <div className="hidden sm:flex">
+      <DynamicSidebar logger={posthog} activeMenu="Bridge" />
```

Comment:
within the sidebar, you are using non-hardcoded values on some elements, due to the portal being 4px-based and the bridge ui being 5px-based (which is the latest system), the spacing is inconsistent on both sites.

see

<img width="336" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/59ff5c89-c41e-4a37-809d-2e9fb37e91e7">
<img width="282" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/475490b8-0876-4395-aeba-03f9f7808140">